jaxrs-client/pom.xml

```diff
@@ -287,6 +287,10 @@
                             <layers>
                                 <layer>cloud-server</layer>
                             </layers>
+                            <!-- Workaround to avoid opentelemetry issue that breaks the Jaxrs applications: removed once WFLY-17955 is integrated -->
```

Comment:
I see that WFLY-17955 is resolved and fix will be on upcoming releases, do we still need these workarounds?

Reply:
I depends on if we target only 28.0.1 or all 28 releases

Reply:
I mean QS will provisioning either 29.0.0 Alpha or 28.0.1 Final, depending on branch, and both seem to have the fix.